app/controllers/application_controller.rb

```diff
@@ -211,6 +222,7 @@ def load_html_blocks
   def authorize_collection
     # skip irrelevant cases
     return unless @collection
+    return if @collection.is_a?(DocumentSet)
```

Comment:
We're willing to break existing document set URLs (so few -- or no -- users), so don't worry about this.